Kitodo/src/main/java/org/kitodo/production/services/data/TaskService.java

```diff
@@ -685,84 +687,26 @@ public List<Task> getPreviousTasksForProblemReporting(Integer ordering, Integer
      * @return list of JSON objects with tasks for specific process id
      */
     List<Map<String, Object>> findByProcessId(Integer id) throws DataException {
-        return findDocuments(getQueryForProcess(id));
+        // TODO delete method stub
+        throw new UnsupportedOperationException("no longer used function");
```

Comment:
Method is not marked as deprecated but is throwing this exception? Why not remove this method and maybe adjust the still used usage places?